tests/test_taling_a_vacation.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+import pytest
+
+from codecademy.taling_a_vacation import hotel_cost
+from codecademy.taling_a_vacation import plane_ride_cost
+from codecademy.taling_a_vacation import rental_car_cost
+from codecademy.taling_a_vacation import trip_cost
+
+@pytest.mark.parametrize("test_hotel_cost_input,test_hotel_cost_expected", [
+    (-sys.maxsize - 1, (-sys.maxsize - 1)*140),
+    (-sys.maxsize, -sys.maxsize*140),
+    (-1, -140),
+    (0, 0),
+    (1, 140),
+    (3037000499, 425180069860),
+    (sys.maxsize - 1, (sys.maxsize-1)*140),
+    (sys.maxsize, sys.maxsize*140)
```

Comment:
Well here is a different story with the hotel.... It's a real life scenario, so the function uses number of days as input to count cost. By definition number of day can't be negative number - otherwise the hotel will have to pay the customer ))))
also think about what will be the maximum value for the upper boundary. It is not sys.maxsize because that is more then the age of the Universe 😄   https://en.wikipedia.org/wiki/Age_of_the_universe
So it should be some reasonable number - i think a person can stay for more than 1 year in the hotel.
So just google about longest hotel stay :)